### DISCUSSION TITLE
Game crashes on startup

### DISCUSSION THREAD

**Discussion by Stone-Red-Code** [voxeltycoon/issues#621] [Troubleshooting] [answered]
**Describe the bug**
The game crashes almost immediately after the "made with Unity" label shows.
This didn't happen yesterday when i played the game.
This is what i see:
![image](https://user-images.githubusercontent.com/56473591/115729934-206ea780-a386-11eb-9831-59ae3c869a7b.png)
 
**Steps to reproduce**
Start the game ¯\_(ツ)_/¯

**What i tried to fix it**
Reinstall the game.
Clearing the data in the AppData folder
Entering the beta branch (I wasn't in it before)

**Build version**
Build ID displayed in Steam: 6582945

**Platform**
Windows 10 Build 19042

**Reproducibility**
Always when i start the game but it only happens on my PC.

**Logs**
[Player.log](https://github.com/voxeltycoon/bug-tracker/files/6358951/Player.log)

**Comment by katepolak** [comment]
Thank you for reporting. From looking at the logs there isn't really anything I can see just with that, other then it most likely being a Unity error rather then a VT specific one, so it will require more investagation from the devs.
One thing that does come into mind though is that possibly some other application is causing some kind of a conflict with Unity, like a antivirus.

**Comment by Stone-Red-Code** [comment]
Ok I'm using a laptop and I noticed that the AV is not the problem it only crashes when my second monitor is not plugged in/is not enabled. It works perfectly if I only enable the second Monitor but as soon as i switch to my built in laptop monitor it instantly crashes. I tried to mess around with the display setting but nothing changed.

**Comment by lassiter55** [comment]
![voxel](https://user-images.githubusercontent.com/83120925/115957512-348be380-a503-11eb-8d4e-67a6211a3f17.jpg)
Spiel startet nicht mehr. bleibt immer hier hängen. AV ist aus, Spieldateien wurden kontrolliert.

Warum startet es nicht?

**Comment by andrewpey** [comment]
@Stone-Red-Code can you please attach the log file from the case when it runs normally (with monitor connected)?

**Comment by Stone-Red-Code** [comment]
@andrewpey
Sure. Here it is:
[Player.log](https://github.com/voxeltycoon/bug-tracker/files/6383260/Player.log)

**Reply by andrewpey** [reply]
Thanks.

**Comment by andrewpey** [comment, marked as answer]
Please try to add the following string:
`force-d3d11-bltblt-mode=`
to `VoxelTycoon_Data\boot.config` file and let us know if this helps.

**Reply by Stone-Red-Code** [reply]
Hey it worked! Thanks.
